terraform/modules/gost_api/grant_digest.tf

```
@@ -0,0 +1,46 @@
+module "grant_digest_scheduled_task" {
+  source  = "../scheduled_ecs_task"
+  enabled = var.enabled && var.enable_grant_digest_scheduled_task
+
+  name_prefix = "${var.namespace}-grant-digest-scheduled-task"
```

This is annoying, but [this error](https://github.com/usdigitalresponse/usdr-gost/actions/runs/8546141230/job/23416072415?pr=2890#step:15:27) in the failing `terraform plan` CI job is due to this `name_prefix` value being too long for certain AWS resources' max character restrictions – the value for the Staging environment, `gost-staging-grant-digest-scheduled-task`, is 40 characters, and the max length for an IAM role prefix is 38 characters ☹️ 

The fix is fortunately easy, albeit not quite as readable as the current solution:
```suggestion
  name_prefix = "${var.namespace}-grant-digest-task"
```
Other preferences for shortening this value (`gost-staging-grant-digest-sched-task` is 37 characters, for example) are also acceptable.